encoding/ipfix/element.go

```diff
@@ -84,154 +75,154 @@ const (
-		ElementSpecifier{22, 0}: FieldDefinition{Uint32, "flowStartSysUpTime"},
+		elementSpecifier{21, 0}: fieldDefinition{Uint32, "flowEndSysUpTime"},
+		elementSpecifier{22, 0}: fieldDefinition{Uint32, "flowStartSysUpTime"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{27, 0}: FieldDefinition{IPv6Address, "sourceIPv6Address"},
-		ElementSpecifier{28, 0}: FieldDefinition{IPv6Address, "destinationIPv6Address"},
-		ElementSpecifier{29, 0}: FieldDefinition{Uint8, "sourceIPv6PrefixLength"},
-		ElementSpecifier{30, 0}: FieldDefinition{Uint8, "destinationIPv6PrefixLength"},
-		ElementSpecifier{31, 0}: FieldDefinition{Uint32, "flowLabelIPv6"},
-		ElementSpecifier{32, 0}: FieldDefinition{Uint16, "icmpTypeCodeIPv4"},
-		ElementSpecifier{33, 0}: FieldDefinition{Uint8, "igmpType"},
+		elementSpecifier{27, 0}: fieldDefinition{IPv6Address, "sourceIPv6Address"},
+		elementSpecifier{28, 0}: fieldDefinition{IPv6Address, "destinationIPv6Address"},
+		elementSpecifier{29, 0}: fieldDefinition{Uint8, "sourceIPv6PrefixLength"},
+		elementSpecifier{30, 0}: fieldDefinition{Uint8, "destinationIPv6PrefixLength"},
+		elementSpecifier{31, 0}: fieldDefinition{Uint32, "flowLabelIPv6"},
+		elementSpecifier{32, 0}: fieldDefinition{Uint16, "icmpTypeCodeIPv4"},
+		elementSpecifier{33, 0}: fieldDefinition{Uint8, "igmpType"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{56, 0}: FieldDefinition{MacAddress, "sourceMacAddress"},
+		elementSpecifier{56, 0}: fieldDefinition{MacAddress, "sourceMacAddress"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{60, 0}: FieldDefinition{Uint8, "ipVersion"},
+		elementSpecifier{60, 0}: fieldDefinition{Uint8, "ipVersion"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{62, 0}: FieldDefinition{IPv6Address, "ipNextHopIPv6Address"},
+		elementSpecifier{62, 0}: fieldDefinition{IPv6Address, "ipNextHopIPv6Address"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{80, 0}: FieldDefinition{MacAddress, "destinationMacAddress"},
-		ElementSpecifier{81, 0}: FieldDefinition{MacAddress, "postSourceMacAddress"},
+		elementSpecifier{80, 0}: fieldDefinition{MacAddress, "destinationMacAddress"},
+		elementSpecifier{81, 0}: fieldDefinition{MacAddress, "postSourceMacAddress"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{176, 0}: FieldDefinition{Uint8, "icmpTypeIPv4"},
-		ElementSpecifier{177, 0}: FieldDefinition{Uint8, "icmpCodeIPv4"},
-		ElementSpecifier{178, 0}: FieldDefinition{Uint8, "icmpTypeIPv6"},
-		ElementSpecifier{179, 0}: FieldDefinition{Uint8, "icmpCodeIPv6"},
+		elementSpecifier{176, 0}: fieldDefinition{Uint8, "icmpTypeIPv4"},
+		elementSpecifier{177, 0}: fieldDefinition{Uint8, "icmpCodeIPv4"},
```

Comment:
Codacy found an issue: [avoid magic numbers like '177', create a named constant for it](https://app.codacy.com/gh/mateumann/clouseau/pullRequest?prid=6383124)